vivarium/core/process.py

```diff
@@ -103,14 +103,6 @@ class can provide a ``defaults`` class variable to specify the
               also contain the following special keys:
 
               * ``name``: Saved to ``self.name``.
-              * ``_original_parameters``: Returned by
-                ``__getstate__()`` for serialization.
-              * ``_no_original_parameters``: If specified with a value
-                of ``True``, original parameters will not be copied
-                during initialization, and ``__getstate__()`` will
-                instead return ``self.parameters``. This puts the
-                responsibility on the user to not mutate process
-                parameters.
```

Comment:
`_original_parameters` no longer needed? What change made this possible?

Reply:
This was part of the convoluted process serialization code I was referring to in my most recent comment. Now that we aren’t implementing our own `__getstate__` and `__setstate__` pickling methods, we no longer have to bother with saving the parameter dictionaries, original or otherwise.